source/requirements/requirements.rst

```diff
@@ -23,6 +23,10 @@ All IBM z/OS collections require these versions of software:
 * `Ansible version`_: 2.9 or later
 * `Python`_: 2.7 or later
 * `OpenSSH`_
+   * Modules which can have Ansible tasks delegated to a localhost need not
+     have OpenSSH enabled. Such modules often rely on a REST API for automation.
+     Refer to the collections under `managed node documentation`_
```

Comment:
I don't see a URL for this link
<img width="607" alt="image" src="https://user-images.githubusercontent.com/25803172/108133653-ca217780-7069-11eb-8ce1-e283fc89e73f.png">

Reply:
This was intended to be just a highlight. Fixed it